packages/components/src/FormField/FormField.test.tsx

```diff
@@ -165,8 +165,9 @@ describe("FormField", () => {
         "The snake which cannot cast its skin has to die. As well the minds which are prevented from changing their opinions; they cease to be mind.";
       const newerValue =
         "They always say time changes things, but you actually have to change them yourself.";
-      const changeHandler = jest.fn();
-
+      const changeHandler = jest.fn(val => {
+        jest.fn(val);
+      });
```

Comment:
do you still need this to be rewritten like this?

Reply:
https://github.com/GetJobber/atlantis/pull/783/commits/223eb5c71a4057f44fdbca1f4f82b6af04d37ee8